download/upgradeRecipe/upgradeRecipe8.adoc

```diff
@@ -121,8 +62,6 @@ In OptaPlanner 8, we have removed most uses of the `Serializable` marker interfa
 https://docs.optaplanner.org/latest/optaplanner-docs/html_single/index.html#integrationWithPersistentStorage[serialize
 via JSON or other formats].
 
-== OptaPlanner Core
```

Comment:
Why removing this (and the next one) header? The idea is to help users navigate through the already long (and growing) document. Not everybody is interested in changes made to Benchmark, for instance.

Reply:
I spent 20 minutes on this. There is no way how to make the header structure logical while maintaining this.
I mean - I could introduce this second level and turn all the individual changes to a third level.
But then, from 8.0.0 to 8.1.0, there would be no third level anymore, the individual changes all on the second level, and the consistency would be gone.

Reply:
Furthermore, as far as I can see, 7.x upgrade recipe didn't do this either.